static/js/stream_color.js

```diff
@@ -103,7 +104,9 @@ function picker_do_change_color(color) {
     const stream_id = Number.parseInt($(this).attr("stream_id"), 10);
     const hex_color = color.toHexString();
     subs.set_color(stream_id, hex_color);
+    popovers.hide_all();
 }
+
```

Comment:
Unrelated code formatting.

Reply:
It is a space after the function in order to make the code look cleaner, every function does have a space after it.